css/styles.css

```
@@ -480,3 +480,280 @@ body._lock {
     text-align: center;
   }
 }
+
+@media (min-width: 70vh) {
```

```suggestion
@media (min-width: 1280px) { 
```
Тут ширина должна быть фиксированной